website/src/pages/Blog/Author/index.tsx

```diff
@@ -0,0 +1,135 @@
+import useStyles from "./styles";
+import { useTranslation } from "react-i18next";
+import {
+  Avatar,
+  Breadcrumbs,
+  Container,
+  Grid,
+  Link,
+} from "@material-ui/core";
+import Footer from "../../../components/Footer";
+import { VIEW_PORT } from "../../../constants";
+import Sponsor from "../../../components/Sponsor";
+import { getAvatar } from "../../../utils/getAvatar";
+import { useViewport } from "../../../hooks/viewportWidth";
+import { pageCount } from "../../../utils/getPageCount";
+import SeoJson from "../../../resources/seo.json";
+import { currentOrigin } from "../../../utils/currentHost";
+import { Metadata } from "../../../components/Metadata";
+import { useAuthorName } from "../../../hooks/extractBlogPath";
+import { Pagination } from "@material-ui/lab";
+import BlogCard from "../../../components/BlogCard";
+import ErrorPage from "../../ErrorPage";
+
+const Blog: React.FC = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const [jsonMdData, setJsonMdData] = useState<any>("");
+  const authorName = useAuthorName();
+  const itemsPerPage = 6;
+  const [page, setPage] = React.useState<number>(1);
+  const { width } = useViewport();
+  const mobileBreakpoint = VIEW_PORT.MOBILE_BREAKPOINT;
+  
+
+  const fetchBlogs = async () => {
+    const { default: blogs } = await import(`../../../posts.json`);
+    setJsonMdData(blogs);
+  };
+
+  useEffect(() => {
+    fetchBlogs();
+  });
+
+  const filteredAuthorData = (jsonMdData || []).filter(
+    (blog: any) => blog.author.toLowerCase().replace(/[^\w ]+/g,'').replace(/ +/g,'-') === authorName
+  );
+
+  const pagination = () => {
+    return (
+      <Pagination
+      count={
+       pageCount(filteredAuthorData)
+      }
+      page={page}
+      onChange={(_event, val) => val? setPage(val) : setPage(1)}
+      className={classes.pagination}
+    />
+    );
+  };
+
+  return (
+    <>
+     <Metadata title={SeoJson.pages.blog.title} description={SeoJson.pages.blog.description} url={`${currentOrigin}${SeoJson.pages.blog.url}`} image={`${currentOrigin}${SeoJson.pages.blog.image}`} isPost={false} type="Series"  />
+        <>
+          <div className={classes.root}>
+            <Container maxWidth="md">
+              {(width > mobileBreakpoint && filteredAuthorData.length > 0) &&
+                <Breadcrumbs aria-label="breadcrumb" className={classes.breadCrumbs}>
+                  <Link color="inherit" href="/blog">
+                    {t('blog.blog')}
+                  </Link>
+                  <Link color="inherit" href={`/blog/author/${authorName}`}>
+                      {filteredAuthorData[0]?.author}
+                  </Link>
+                </Breadcrumbs>
+              }
+              <div className={classes.authorWrapper}>
+                <Avatar
+                  alt={filteredAuthorData[0]?.author}
+                  src={`/images/blog/authors/${getAvatar(filteredAuthorData[0]?.author || authorName)}.png`}
```

Comment:
Does this work ? Have you verified it ?

Reply:
Yes. I have attached a screenshot in PR description with name `John doe`